sdk/storage/storage-blob/src/BlobLeaseClient.ts

```diff
@@ -17,45 +17,46 @@ import { BlobClient } from "./BlobClient";
  */
 export interface Lease {
   /**
-   * @member {string} [etag] The ETag contains a value that you can use to
+   * The ETag contains a value that you can use to
    * perform operations conditionally. If the request version is 2011-08-18 or
    * newer, the ETag value will be in quotes.
    */
   etag?: string;
   /**
-   * @member {Date} [lastModified] Returns the date and time the container was
+   * Returns the date and time the container was
    * last modified. Any operation that modifies the blob, including an update
    * of the blob's metadata or properties, changes the last-modified time of
    * the blob.
    */
   lastModified?: Date;
   /**
-   * @member {string} [leaseId] Uniquely identifies a container's lease
+   * Uniquely identifies a container's lease
    */
   leaseId?: string;
   /**
-   * @member {number} [leaseTime] Approximate time remaining in the lease
+   * Approximate time remaining in the lease
    * period, in seconds.
    */
   leaseTime?: number;
   /**
-   * @member {string} [requestId] This header uniquely identifies the request
+   * This header uniquely identifies the request
    * that was made and can be used for troubleshooting the request.
    */
   requestId?: string;
   /**
-   * @member {string} [version] Indicates the version of the Blob service used
+   * Indicates the version of the Blob service used
    * to execute the request. This header is returned for requests made against
    * version 2009-09-19 and above.
    */
   version?: string;
   /**
-   * @member {Date} [date] UTC date/time value generated by the service that
+   * UTC date/time value generated by the service that
    * indicates the time at which the response was initiated
    */
   date?: Date;
   /**
-   * @member {string} [errorCode]
+   * Error code if any associated with the response that returned
+   * the Lease information.
```

Comment:
@XiaoningLiu,
I guessed the description here, so if you have suggestions on a better text, please do suggest

Reply:
Looks good.